backup-restore/neo4j/cluster/neo4j-cluster-backup-restore-template.yaml

```diff
@@ -0,0 +1,67 @@
+apiVersion: spp-data-protection.isf.ibm.com/v1alpha1
+kind: Recipe
+metadata:
+  name: neo4j-cluster-backup-restore-recipe
+  namespace: ibm-spectrum-fusion-ns
+spec:
+  appType: neo4j
+  groups:
+    - name: neo4j-volumes
+      type: volume
+    - name: neo4j-resources
+      type: resource
+      excludedResourceTypes:
+        - pods
+  hooks:
+  - name: neo4j-core-statefulset-check
+    type: check
+    namespace: ${GROUP.neo4j-resources.namespace}
+    selectResource: statefulset
+    labelSelector: app.kubernetes.io/component=core
+    timeout: 120
+    onError: fail
+    chks:
+    - name: replicasReady
+      timeout: 600
+      onError: fail
+      condition: "{$.spec.replicas} == {$.status.readyReplicas}"
+  - name: neo4j-replica-statefulset-check
+    type: check
+    namespace: ${GROUP.neo4j-resources.namespace}
+    selectResource: statefulset
+    labelSelector: app.kubernetes.io/component=replica
+    timeout: 120
+    onError: fail
+    chks:
+    - name: replicasReady
+      timeout: 600
+      onError: fail
+      condition: "{$.spec.replicas} == {$.status.readyReplicas}"      
+  - name: neo4j-pod-exec
+    type: exec
+    namespace: ${GROUP.neo4j-resources.namespace}
+    nameSelector: $REPLACE_WRITER_POD_NAME
+    timeout: 120
+    onError: fail
+    ops:
+    - name: disable_write
+      command: "cypher-shell -u neo4j -p $NEO4J_SECRETS_PASSWORD \"CALL dbms.setConfigValue('dbms.databases.default_to_read_only', 'true');\""
```

Comment:
i think this will put password in plain text in recipe file, then same will go in logfile
can you explore APIs to get the password from secret?

Reply:
CURL command was not there in the POD